webui_runner/plugs/lucet_app/download.sh

```diff
@@ -14,10 +14,7 @@ git reset --hard
 git submodule init && git submodule update
 git submodule foreach --recursive git submodule init
 git submodule foreach --recursive git submodule update
+git pull
```

Comment:
Is this necessary? If so, we should document why. Also, are lines 15-16 also necessary or could a `--recursive` on line 14 be used instead?

Reply:
Agree, there is some redundancy here .. though redundancy is short-circuited (things aren't really updated twice) . Initially, I had a more minimal set of instructions (probably just line 14 with --recursive as you point out) but I found it did not always work to put the code in the state I expected. I assumed I didn't understand some nuance here but doing things redundantly with foreach seemed to work which was the only goal at the time. I'll try to remove line 14 here.

Reply:
Note, like some other suggestions because templates are mostly identical this change isn't specific to getting lucet to build. Probably should be in a different patch addressing all the git pulls.

Reply:
Ok, I opened https://github.com/bytecodealliance/sightglass/issues/32 for that.